src/utils/dashboardModelUtils.ts

```diff
@@ -0,0 +1,142 @@
+import { RootNodeModel, SectionNodeModel, WidgetModel } from '../interfaces/nodeModels';
+
+
+export function removeWidgetFromState(contents: any, id: string) {
```

Comment:
Also I believe you can change the type of contents here to this (and import accordingly)
```suggestion
export function removeWidgetFromState(contents: DashboardContentsModel, id: string) {
```